StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/UpdateCardViewControllerSnapshotTests.swift

```diff
@@ -28,14 +28,31 @@ final class UpdateCardViewControllerSnapshotTests: STPSnapshotTestCase {
         let sut = UpdateCardViewController(paymentOptionCell: .init(frame: .zero),
                                            paymentMethod: STPFixtures.paymentMethod(),
                                            removeSavedPaymentMethodMessage: "Test removal string",
-                                           appearance: appearance)
+                                           appearance: appearance,
+                                           isTestMode: false)
         let testWindow = UIWindow(frame: CGRect(x: 0, y: 0, width: 428, height: 500))
         testWindow.isHidden = false
         if darkMode {
             testWindow.overrideUserInterfaceStyle = .dark
         }
         testWindow.rootViewController = sut
-        sut.view.autosizeHeight(width: 375)
+        sut.view.autosizeHeight(width: 375, height: 280)
         STPSnapshotVerifyView(sut.view)
     }
 }
+
+private extension UIView {
+    /// Constrains the view to the given width and autosizes its height.
+    /// - Parameter width: Resizes the view to this width
+    func autosizeHeight(width: CGFloat, height: CGFloat) {
```

Comment:
Can we update the comment and name here to reflect that we're autosizing to a fixed width & height?